daisy_workflows/image_build/debian/debian_13_fai.wf.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Name": "build-debian-13-fai",
+  "Vars": {
+    "build_date": {"Value": "${DATE}", "Description": "Build datestamp used to version the image."},
+    "publish_project": {
+      "Value": "${PROJECT}",
+      "Description": "A project to publish the resulting image to."
+    }
+  },
+  "Steps": {
+    "build": {
+      "Timeout": "60m",
+      "IncludeWorkflow": {
+        "Path": "./debian_fai.wf.json",
+        "Vars": {
+          "build_date": "${build_date}",
+          "debian_version": "sid"
+        }
+      }
+    },
+    "copy-gcs-object": {
+      "CopyGCSObjects": [
+        {
+          "Source": "${OUTSPATH}/root.tar.gz",
+          "Destination": "${OUTSPATH}/root.tar.gz"
+        }
+      ]
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "debian-13-v${build_date}",
+          "RawDisk": {
+            "Source": "${OUTSPATH}/root.tar.gz"
+          },
+          "Description": "Debian, Debian GNU/Linux, 13 (sid), amd64 built on ${build_date}",
```

Comment:
trixie?

Reply:
Oh yes I missed this reference, thanks. Updated